packages/e2e-tests/src/specs/dashboard/settings/adminUser/customFonts.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import {
+  withExperimentalFeatures,
+  visitSettings,
+  addCustomFont,
+  removeCustomFont,
+  getFontList,
+  getSelectedFont,
+} from '@web-stories-wp/e2e-test-utils';
+
+const FONT_BASE_URL = `${process.env.WP_BASE_URL}/wp-content/e2e-assets`;
+const OPEN_SANS_CONDENSED_LIGHT = 'Open Sans Condensed Light';
+const OPEN_SANS_CONDENSED_LIGHT_URL = `${FONT_BASE_URL}/OpenSansCondensed-Light.ttf`;
+const OPEN_SANS_CONDENSED_BOLD = 'Open Sans Condensed Bold';
+const OPEN_SANS_CONDENSED_BOLD_URL = `${FONT_BASE_URL}/OpenSansCondensed-Bold.ttf`;
+const OPEN_SANS_CONDENSED_LIGHT_ITALIC_URL = `${FONT_BASE_URL}/OpenSansCondensed-LightItalic.ttf`;
+
+const findByUrl = (arr, val) => {
+  const found = arr.filter((o) => o.url === val);
+  if (found) {
+    return found[0];
+  }
+  return [];
+};
+
+const removeAllFonts = async () => {
+  await visitSettings();
+  let fonts = await getFontList();
+
+  if (fonts.length === 0) {
+    return;
+  }
+
+  for (let i = 0; i < fonts.length; i++) {
+    // eslint-disable-next-line no-await-in-loop -- async for
+    await removeCustomFont(fonts[i].name);
+  }
+
+  await page.waitForTimeout(500);
+  fonts = await getFontList();
+  expect(fonts).toHaveLength(0);
+};
+
+describe('Custom Fonts', () => {
+  withExperimentalFeatures(['customFonts']);
+
+  beforeEach(async () => {
+    await visitSettings();
+    await removeAllFonts();
+  });
```

Comment:
```suggestion
  beforeEach(async () => {
    await visitSettings();
 });
    
 beforeAll(async () => {   
    await removeAllFonts();
 });
```

We don't see to do this before each test. Reset back to no font at start and after each test. That should be enough